clickhouse/columns/date.cpp

```diff
@@ -113,4 +113,69 @@ ItemView ColumnDateTime::GetItem(size_t index) const {
     return data_->GetItem(index);
 }
 
+
+ColumnDateTime64::ColumnDateTime64()
+    : Column(Type::CreateDateTime64(3ul))
```

Comment:
Maybe we shouldn't provide a default constructor at all or come up with a different default precision. I will be happy to get feedback regarding this line.